src/transforms/operationTransforms.ts

```diff
@@ -104,9 +104,10 @@ export function transformOperationSpec(
 
   const hasMultipleRequests = operationDetails.requests.length > 1;
   for (const request of operationDetails.requests) {
-    const isXML = request.mediaType
-      ? request.mediaType === KnownMediaType.Xml
-      : operationDetails.mediaTypes.has(KnownMediaType.Xml);
+    const isXML =
+      operationDetails.mediaTypes.has(KnownMediaType.Xml) ??
+      (request.mediaType && request.mediaType === KnownMediaType.Xml);
```

Comment:
Comment from Jose: Check the code without line 109. If there is any change, then leave it. Else, remove that line

Reply:
Checked and removed that line